src/components/FiltersPanel/FiltersPanel.tsx

```diff
@@ -180,7 +180,11 @@ export const FiltersPanel: FC<{
                             <FiltersCounter total={total} counter={counter} />
                         </FiltersCounterContainer>
                         <FiltersMenuContainer>
-                            <FiltersMenuItem ref={filterNodeRef} onClick={() => setFilterVisible((p) => !p)}>
+                            <FiltersMenuItem
+                                active={!isFilterStateEmpty(queryState)}
```

Comment:
This is including starred & watching filter flags, I don't think that it's need here

Reply:
is should be in other PR, sorry